github/resource_github_repository_tag_protection.go

```diff
@@ -98,6 +98,13 @@ func resourceGithubRepositoryTagProtectionRead(d *schema.ResourceData, meta inte
 		}
 		return err
 	}
+
```

Comment:
Wouldn't we also want to do this if the call to `client.Repositories.ListTagProtection` 404s? What happened when you tested this?

Reply:
Returns [], worked as expected.